canonical-web-and-design/ubuntu.com
# Remove /desktop/activedirectory redirect

## Done

- Remove /desktop/activedirectory redirect
- Redirect wasn't published, so removed, plan to add it to help.ubuntu.com instead


## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/desktop/activedirectory
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that http://0.0.0.0:8001/desktop/activedirectory no longer redirects


## Issue / Card

Fixes #8288
